spec/furniture/marketplace/delivery_areas_system_spec.rb

```diff
@@ -9,6 +9,26 @@
     sign_in(space.members.first, space)
   end
 
+  describe "Restoring Delivery Areas" do
+    let!(:delivery_area) do
+      create(:marketplace_delivery_area, :archived, marketplace:,
+        label: "Oakland", price_cents: 10_00)
+    end
+
+    it "Makes the DeliveryArea selectable" do
+      visit(polymorphic_path(marketplace.location(child: :delivery_areas)))
+      click_link("Archived Delivery Areas")
+      within("##{dom_id(delivery_area)}") do
+        click_link("Edit")
```

Comment:
Is there any chance delivery area data persists/leaks across cases so you'd want to do `click_link("Edit", :match => :first)` etc...?

Reply:
The `within('##{dom_id(delivery_area)}') do ... end` scopes this click_link to child elements of the DOM element with the delivery area's ID. This should mitigate leakage across test case runs.

That said, leakage across test case runs are very unlikely to the point of "not worth worrying about" in most rails apps using RSpec because of [rspec's transactional fixtures feature](https://github.com/zinc-collective/convene/blob/main/spec/rails_helper.rb#L40).

Reply:
OH right right I see that scope now. Coolio.